UI/Web/src/app/manga-reader/_components/infinite-scroller/infinite-scroller.component.ts

```diff
@@ -232,6 +239,26 @@ export class InfiniteScrollerComponent implements OnInit, OnChanges, OnDestroy,
       takeUntilDestroyed(this.destroyRef)
     );
 
+
+    this.widthSliderValue$ = this.readerSettings$.pipe(
+      map(values => (parseInt(values.widthSlider) <= 0) ? '' : values.widthSlider + '%'),
+      takeUntilDestroyed(this.destroyRef)
+    );
+
+    //perfom jump so the page stays in view
+    this.widthSliderValue$.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+      this.currentPageElem= this.document.querySelector('img#page-' + this.pageNum);
+      if(!this.currentPageElem) return
+      var images = Array.from(document.querySelectorAll('img[id^="page-"]')) as HTMLImageElement[];
+      for(let i = 0; i < images.length; i++) {
```

Comment:
You probably just want to use a foreach loop so we don't need ` var img = images[i];`